### DISCUSSION TITLE
More Content Season of AI?

### DISCUSSION THREAD

**Discussion by nielsberglund** [microsoft/community-content#4] [Season of AI]
Hi all!

Will there be more content for Season of AI coming after Build? We are planning a full-day event around Season of AI: [**Data & AI Community Day Durban: Season of AI**](https://www.meetup.com/azure-transformation-labs/events/300754170/), and I just wonder how much content will be available.

We'll in any case add other content but - as I said - interesting to know.

Niels

**Comment by Tbush-dev** [comment]
@nielsberglund - The two topics through August are Generative AI for Developers (.NET) and Getting Started with Azure AI Studio (General).  Let us know what else you you'd like to see included in the assets - more sample code, slides, guidance for demos, etc.

**Reply by nielsberglund** [reply]
I had assumed (wrongly) that there would be more than two topics, so I would like to have seen more AI topics. Based on that, we planned to run a full-day Season of AI event with predominantly AI topics. We'll just change the tack, the event will still be themed Season of AI, but we'll have other topics as well. No problem!

**Comment by toddanglin** [comment]
@nielsberglund As @Tbush-dev shared, these will likely be the only two "official" prepared sessions we share here for this season, but we/Microsoft _will_ have other AI-related content/sessions we can share after Build if you want to reuse as part of your Data & AI Day. Shoot me an email and I can have some of our AI and Data Advocates share more of what's already available for reuse today: todd.anglin [at] microsoft.

**Reply by nielsberglund** [reply]
Hi @toddanglin!

Cool stuff, I'll drop you an email. Thanks!
